src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -3,17 +3,39 @@
 import com.revrobotics.CANSparkLowLevel.MotorType;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkAbsoluteEncoder;
+import com.revrobotics.SparkPIDController;
 
 public class ArmSubsystem extends SubsystemBase {
-     /** Creates a new FlyWheelSubsystem. */
+     /** Creates a new ArmSubsystem. */
```

Comment:
Is the `ArmSubsystem` different than the `PivotSubsystem`?

Reply:
yes ':)